WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -320,6 +320,65 @@
 
     </ScrollView>
 
+    <LinearLayout
+        android:id="@+id/no_site_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="@dimen/my_site_no_site_view_margin"
+        android:gravity="center"
+        android:orientation="vertical">
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:contentDescription="@string/my_site_no_sites_view_drake"
+            android:src="@drawable/my_site_no_sites_drake" />
+
+        <org.wordpress.android.widgets.WPTextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_large"
+            android:gravity="center"
+            android:text="@string/my_site_no_sites_view_title"
+            android:textColor="@color/grey_dark"
+            android:textSize="@dimen/text_sz_double_extra_large"
+            app:fontVariation="light" />
+
+        <org.wordpress.android.widgets.WPTextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/my_site_no_site_view_subtitle_margin_top"
+            android:gravity="center"
+            android:text="@string/my_site_no_sites_view_subtitle"
+            android:textColor="@color/grey"
+            android:textSize="@dimen/my_site_no_site_view_subtitle_font"
+            app:fontVariation="light" />
+
+        <android.support.v7.widget.CardView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/my_site_no_site_view_button_margin_top"
+            card_view:cardBackgroundColor="@color/blue_medium"
+            card_view:cardCornerRadius="@dimen/my_site_no_site_view_button_corner_radius"
+            card_view:cardElevation="@dimen/card_elevation">
+
+            <org.wordpress.android.widgets.WPTextView
+                android:id="@+id/my_site_add_site_btn"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="?android:attr/selectableItemBackground"
+                android:paddingBottom="@dimen/margin_medium"
+                android:paddingLeft="@dimen/margin_large"
+                android:paddingRight="@dimen/margin_large"
+                android:paddingTop="@dimen/margin_medium"
+                android:text="@string/my_site_no_sites_view_button_title"
```

Comment:
Can we avoid the extra string resource and just reuse `@string/site_picker_add_site` here, with `android:textAllCaps="true"`?